add version 3.8.2 to r-gtools

glennpj opened the pull request.

glennpj committed:
add version 3.8.2 to r-gtools

adamjstewart commented on var/spack/repos/builtin/packages/r-gtools/package.py:
```diff
@@ -37,12 +38,12 @@ class RGtools(RPackage):
       match ('smartbind')
     - generate significance stars from p-values ('stars.pval')
     - convert characters to/from ASCII codes.
```
Spack removes newline characters from the description when running `spack info`, so this won't format well. Maybe add semicolons at the end of each bullet point?

glennpj replied:
I added semicolons to the end of each list item as well as converted the '-'s to '[#]s'.

glennpj committed:
Improve formatting of description

In case the list gets formatted as a non-list:
- added semicolons to end of list items
- replaced dashes with [#]
